src/dmd/expression.d

```diff
@@ -5307,6 +5307,49 @@ extern (C++) final class DotVarExp : UnaExp
         if (e1.op == TOK.this_)
             return var.checkModify(loc, sc, e1, flag);
 
+        // Issue 12764 : https://issues.dlang.org/show_bug.cgi?id=12764
```

Comment:
Please delete "Issue 12764 :" as it is redundant with the URL. In general, comments should not contain non-essential verbiage.